### DISCUSSION TITLE
How to add untranslatable words to vocabulary

### DISCUSSION THREAD

**Discussion by Rombersoft** [WeblateOrg/weblate#9276] [Q&A]
I have sentences which contain untranslatable words but in several languages these words were translated.
For example, I have the following sentence:
The next year Microsoft will buy the Sunshine company.
I want that weblate to translate this sentence but the words 'Microsoft' and 'Sunshine' would left unchanged.
Does weblate have such possibility?

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/user/glossary.html#untranslatable-terms

**Comment by Rombersoft** [comment]
I don't want to mark the whole sentence as untranslatable terms. From the whole sentence, I want to mark just one word. How to do it?

**Reply by nijel** [reply]
Add just that one word to glossary
